modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -46,9 +46,13 @@ spec:
         updateStrategy: <2>
           registryPoll:
             interval: 1h
+      status:
+        connectionState:
+            lastObservedState: READY <3>
 ----
 <1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed.
 <2> Set how frequently the Operator Lifecycle Manager (OLM) polls the index image for new Operator versions with the `registryPoll.interval` field. This change is not needed if a new index image tag is always pushed for y-stream and z-stream Operator updates. The `registryPoll.interval` field can be set to a shorter interval to expedite the update, however shorter intervals increase computational load. To counteract this, you can restore `registryPoll.interval` to the default value once the update is complete.
+<3> To successfully finish updates, {cgu-operator} utilizes the `status.connectionState.lastObserved: READY` field.
```

Comment:
```suggestion
<3> To successfully finish updates, {cgu-operator} uses the `status.connectionState.lastObserved: READY` field.
```
Per IBM Style Word Usage, do not use the word "utilize," and instead use a simpler word, such as "use".